_gsocblogs/2022/blog_Rucio_NimishBongale.md

```diff
@@ -1,127 +1,279 @@
+
+- [Modify public folder as per latest web standards](https://github.com/rucio/webui/pull/80)
+
+- [Replaced localStorage with sessionStorage, "any" with "unknown"](https://github.com/rucio/webui/pull/76)
+
+- [Removed .env, package-lock.json .gitignore entry](https://github.com/rucio/webui/pull/69)
+
+- [OIDC POC Complete](https://github.com/rucio/webui/pull/68)
+
+- [Keyboard navigation, accessibility, miscellaneous fixes](https://github.com/rucio/webui/pull/65)
+
+- [Feature 52 alert notifications](https://github.com/rucio/webui/pull/55)
+
+- [Feature 53 initial auth workflow](https://github.com/rucio/webui/pull/54)
+
+- [Feature 47 login unit test cases](https://github.com/rucio/webui/pull/51)
+
+- [Create REST API wrapper utility](https://github.com/rucio/webui/pull/50)
+
+- [Feature 46 add org logos](https://github.com/rucio/webui/pull/49)
+
+- [Frontend Login Infrastructure complete](https://github.com/rucio/webui/pull/45)
+
+- [Feature 37 Implement environment layer, add OIDC configuration, other cleanup](https://github.com/rucio/webui/pull/43)
+
+- [Feature 38 client side routing](https://github.com/rucio/webui/pull/40)
+
+- [Feature 36 CI for React app](https://github.com/rucio/webui/pull/39)
+
+- [Feature 10 login page](https://github.com/rucio/webui/pull/35)
+
+- [Feature 10 login page components](https://github.com/rucio/webui/pull/34)
+
+- [Custom CI/CD pipeline replacing third party gh action](https://github.com/rucio/webui/pull/33)
 
 - [Feature 13 custom cicd replace third party gh action](https://github.com/rucio/webui/pull/32)
 
-#32 by [nimishbongale](https://github.com/rucio/webui/issues?q=is%3Apr+author%3Animishbongale) was closed on 11 Jun
+- [Initial Setup of Storybook, React App, CICD deploy to GitHub pages](https://github.com/rucio/webui/pull/16)
 
-- [Initial Setup of Storybook, React App, CICD deploy to GitHub pages](https://github.com/rucio/webui/pull/16) [DevOps](https://github.com/rucio/webui/issues?q=is%3Apr+author%3Animishbongale+label%3ADevOps) [gsoc22](https://github.com/rucio/webui/issues?q=is%3Apr+author%3Animishbongale+label%3Agsoc22)
+### Key Takeaways
 
-#16 by [nimishbongale](https://github.com/rucio/webui/issues?q=is%3Apr+author%3Animishbongale) was merged on 10 Jun• [Approved](https://github.com/rucio/webui/pull/16#partial-pull-merging) [1.0.0](https://github.com/rucio/webui/milestone/1)
+- Having the right mentors are key to any project - a huge shoutout to @Mayank, @Martin & @Cedric! (special mention to @Valentina for parterning with me on this project!)
+- Creating an OSS project from scratch having a huge impact has been a dream of mine, and I think I very well might've done it!
+- Rethinking UI really requires you to step out of your developer shoes and don the product hat.
+- Being passionate about something really drives you to do it consistently.
+- While the GSoC period ends here, contributing to OSS should continue!
```

Comment:
Maybe you could also add a section to summarize the pending tasks and what could be improved :)